osm_to_sandbox/osm_to_sandbox.py

```diff
@@ -371,5 +395,41 @@ def main():
     print('Done.')
 
 
+def cli():
+    parser = argparse.ArgumentParser(
+        description="Downloads data from Overpass API and uploads it to the mapping "
+                    "sandbox.",
+        epilog="Because sandboxes are for grown-ups, too!",
+    )
+    parser = add_args(parser)
+    args = parser.parse_args()
+    args.bbox = [float(x.strip()) for x in args.bbox.split(',')]
+    main(args.bbox, args.auth_header, args.overpass_api)
+
+
+def add_args(parser):
+    parser.add_argument(
+        "bbox",
+        help="The target bounding box in format minlon,minlat,maxlon,maxlat. "
+             "Get the bounding box from https://boundingbox.klokantech.com/.",
+    )
+    parser.add_argument("-auth",
+                        required=True,
+                        help="This flag will spawn a password prompt before entering "
+                             "the program. Authentication is necessary to upload data "
+                             "to the sandbox.",
+                        dest='auth_header',
+                        action=AuthPromptAction,
+                        type=str)
+    parser.add_argument("--overpassURL",
```

Comment:
Just `--overpass` would be better IMO.

Reply:
renamed in https://github.com/Zverik/osm_to_sandbox/commit/ba12e74e963b04d9aa18093cbd5e765cae4e7ac1